packages/web/app/pages/[orgId]/view/settings.tsx

```diff
@@ -38,30 +33,68 @@ import {
 import { useRouteSelector, useToggle } from '@/lib/hooks';
 import { withSessionProtection } from '@/lib/supertokens/guard';
 
+const Integrations_CheckIntegrationsQuery = graphql(`
+  query Integrations_CheckIntegrationsQuery($selector: OrganizationSelectorInput!) {
+    organization(selector: $selector) {
+      organization {
+        id
+        viewerCanManageOIDCIntegration
+        ...OIDCIntegrationSection_OrganizationFragment
+        id
+        hasSlackIntegration
+        hasGitHubIntegration
+      }
+    }
+    isGitHubIntegrationFeatureEnabled
+  }
+`);
+
+const DeleteSlackIntegrationMutation = graphql(`
+  mutation Integrations_DeleteSlackIntegration($input: OrganizationSelectorInput!) {
+    deleteSlackIntegration(input: $input) {
+      organization {
+        hasSlackIntegration
+      }
```

Comment:
I think urql needs `id` to update the cache entry, no?

Reply:
ur right